tests/opentitan/opentitan-current/module_tests/prim_subst_perm/prim_subst_perm.vlt

```diff
@@ -1,5 +1,10 @@
 `verilator_config
 
+// Warnings are thrown only by uhdm-verilator
+// because Surelog ignores "verilator split_var" comment
+// in declaration of data_state variable 
+// https://verilator.org/guide/latest/extensions.html#cmdoption-verilator-split_var
+
```

Comment:
Can we then use a [`split_var` config option](https://verilator.org/guide/latest/exe_verilator.html#cmdoption-arg-11) to avoid those? This can help avoid the performance impact of `UNOPTFLAT`.

Reply:
This option works only for `UNOPTFLAT`s. I will investigate why there are also `ALWCOMBORDER`s in https://github.com/chipsalliance/UHDM-integration-tests/pull/643